ComponentKit/Utilities/CKComponentDelegateAttribute.mm

```diff
@@ -30,16 +30,17 @@ CKComponentViewAttributeValue CKComponentDelegateAttribute(SEL selector,
     return {
       {
         std::string("Delegate-noop-") + sel_getName(selector) + "-",
-        ^(UIView *view, id value) {}, ^(UIView *view, id value) {}
+        ^(UIView *, CK::ViewAttribute::BoxedValue) {},
+        ^(UIView *, CK::ViewAttribute::BoxedValue) {}
       },
-      @YES  // Bogus value, we don't use it.
+      nil  // Bogus value, we don't use it.
     };
   }
 
   return {
     {
       std::string(sel_getName(selector)) + CKIdentifierFromDelegateForwarderSelectors(selectors),
-      ^(UIView *view, id value){
+      ^(UIView *view, CK::ViewAttribute::BoxedValue){
```

Comment:
We definitely can, but this shouldn't be used in any user-code anyways. If a user is defining their own property, it should look something like:

```
const CKComponentViewAttributeValue customViewAttribute = {
  "custom-view-attribute",
  ^(CustomView *view, CustomType value) {
    view.customValue = value;
  }
};
```

Do you still think it's worth exposing this non-namespaced?